lambda-buffers-frontend/src/LambdaBuffers/Frontend/PPrint.hs

```diff
@@ -0,0 +1,60 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module LambdaBuffers.Frontend.PPrint () where
```

Comment:
I'd include the instances with the declarations - it's nicer for people reading source code.

Reply:
Too much code imo, and this is the main module for 'formatting' actions and I kind a want to treat it separately.